engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/MethodDefinitionExtractor.scala

```diff
@@ -126,7 +126,14 @@ object MethodDefinitionExtractor {
         case param: Parameter =>
           val foundParam = prepareValue(param.name).getOrElse(throw new IllegalArgumentException(s"Missing parameter: ${param.name}"))
           //FIIIXME??
-          validateType(param.name, foundParam, param.originalType.objType.klass)
+          val klass = param.originalType match {
+            case s: ScalarTypingResult =>
+              s.objType.klass
+            case _ =>
```

Comment:
for union at least one validateType should pass => we can remove FIIIXME then ;)